source/Templates/NewStep/Sashimi.NamingIsHard/source/Sashimi.Tests/MyActionHandlerFixture.cs

```diff
@@ -0,0 +1,21 @@
+using Calamari.NamingIsHard;
+using NUnit.Framework;
+using Sashimi.Tests.Shared.Server;
+
+namespace Sashimi.NamingIsHard.Tests
+{
+    [TestFixture]
+    public class MyActionHandlerFixture
+    {
+        [Test]
+        public void Test1()
+        {
+            ActionHandlerTestBuilder.CreateAsync<MyActionHandler, Program>()
```

Comment:
Make this test as "real" as the Calamari one, this should be the default test to write

Reply:
Fixed